kpops/components/streams_bootstrap/streams/model.py

```diff
@@ -52,10 +53,23 @@ def dict(
 
 
 class StreamsAppAutoScaling(BaseModel):
-    consumergroup: str
-    topics: list[str] = []
+    enabled: bool = Field(
+        default=False, description="Whether to enable auto-scaling using KEDA."
+    )
+    consumer_group: str = Field(
+        title="Consumer group",
+        description="Name of the consumer group used for checking the offset on the topic and processing the related lag.",
+    )
+    lag_threshold: int = Field(
+        title="Lag threshold",
+        description="Average target value to trigger scaling actions.",
+    )
+    topics: list[str] = Field(
+        default=[],
+        description="List of auto-generated Kafka Streams topics used by the streams app.",
+    )
```

Comment:
should we add the other attributes as well?
https://github.com/bakdata/streams-bootstrap/tree/master/charts/streams-app

Reply:
Sounds good. I don't think anything will break. Only down-side is that KPOps could become harder to use with older/newer chart versions if we are too strict